index.bs

```diff
@@ -2365,49 +2365,99 @@ Reviews and Review Responsibilities</h4>
 <h5 id="wide-review">
 Wide Review</h5>
 
-	The requirements for <dfn id=dfn-wide-review export>wide review</dfn> are not precisely defined by the W3C Process.
-	The objective is to ensure that the entire set of stakeholders of the Web community,
-	including the general public,
-	have had adequate notice of the progress of the [=Working Group=]
-	(for example through notices posted to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>)
-	and were able to actually perform reviews of and provide comments on the specification.
-	A second objective is to encourage groups to request reviews
-	early enough that comments and suggested changes
-	can still be reasonably incorporated in response to the review.
-	Before approving transitions,
-	the [=Director=] will consider who has been explicitly offered
-	a reasonable opportunity to review the document,
-	who has provided comments,
-	the record of requests to and responses from reviewers,
-	especially <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">W3C Horizontal Groups</a> [[CHARTER]]
-	and groups identified as dependencies in the charter
-	or identified as <a href="https://www.w3.org/2001/11/StdLiaison.html">liaisons</a> [[LIAISON]],
-	and seek evidence of clear communication to the general public
-	about appropriate times and which content to review
-	and whether such reviews actually occurred.
+	The objective of <dfn id=dfn-wide-review export>wide review</dfn> is to seek
+	feedback on a specification from the entire web community, including the general
+	public. The goal is the early identification and resolution of issues in order to make the specification as robust as possible before it becomes a [=W3C Recommendation=]. 
 
-	For example,
-	inviting review of new or significantly revised sections published in Working Drafts,
-	and tracking those comments
-	and the [=Working Group=]'s responses,
-	is generally a good practice which would often be considered positive evidence of wide review.
-	[=Working Groups=] <em class="rfc2119">should</em> announce to other W3C Working Groups
-	as well as the general public,
-	especially those affected by this specification,
-	a proposal to enter [=Candidate Recommendation=]
-	(for example in approximately 28 days).
-	By contrast a generic statement in a document
-	requesting review at any time
-	is likely not to be considered as sufficient evidence
-	that the group has solicited wide review.
-
-	A [=Working Group=] could present evidence that wide review has been received,
-	irrespective of solicitation.
-	But it is important to note that receiving many detailed reviews
-	is not necessarily the same as wide review,
-	since they might only represent comment
-	from a small segment of the relevant stakeholder community.
```

Comment:
Seems like losing the sense of this section is a shame; I thought this process text was very useful in framing how WGs think about their wide review. I wonder if, in the way that @frivoal suggested in https://github.com/w3c/w3process/pull/401/files#r426429326 this can be refactored somehow so that the definition of documentation for demonstrating due process has been applied to review comments can be moved to a place where it clearly applies both to all wide review comments including horizontal review comments.

Reply:
Moving it makes sense. I'm just not sure of the mechanics of doing it in a way that does not get lost. if @frivoal  can make it happen, since his Github fu is likely better than mine) that would e good.